e2e/config_fields.go

```diff
@@ -0,0 +1,27 @@
+package e2e
+
+import "fmt"
+
+type TestEnvironment int
+
+const (
+	// Local environment uses a local memstore analog for the actual eigenDA network
+	Local TestEnvironment = iota + 1
+	// Testnet runs tests against holesky testnet
+	Testnet
+)
```

Comment:
Do these do anything other than setting up the backend (memstore vs eigenda-holesky)? If not, could we use that explicit vocabulary (backend) instead of "Environment", which feels like another layer of uncertainty for test readers/maintainers.

Reply:
Renamed [657ada3](https://github.com/Layr-Labs/eigenda-proxy/pull/337/commits/657ada33d9cce2ae0f383ccd259ff55aae809103)

Reply:
tip: golang enums are super dumb, so best to name the options with the type as suffix, like `MemstoreTestBackend`

Reply:
Good to know. Enum is gone in most recent iteration, though